git/git.go

```diff
@@ -66,8 +66,8 @@ const (
 	DefaultBranch = "master"
 
 	defaultGithubAuthRoot = "git@github.com:"
-	defaultGitUser        = "Anago GCB"
-	defaultGitEmail       = "nobody@k8s.io"
+	defaultGitUser        = "Kubernetes Release Robot"
+	defaultGitEmail       = "k8s-release-robot@users.noreply.github.com"
```

Comment:
Let's change this for now but these are the kubernetes-specific bits we should move out of `release-sdk` and define them in `k/release`.